airflow/migrations/versions/0118_2_4_0_add_last_modified_at_column_to_dagrun_.py

```diff
@@ -0,0 +1,55 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""add last_modified_at column to DagRun and TaskInstance
+
+Revision ID: ee8d93fcc81e
+Revises: ecb43d2a1842
+Create Date: 2022-09-08 19:08:37.623121
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+from airflow.migrations.db_types import TIMESTAMP
+
+# revision identifiers, used by Alembic.
+revision = 'ee8d93fcc81e'
+down_revision = 'ecb43d2a1842'
+branch_labels = None
+depends_on = None
+airflow_version = '2.4.0'
```

Comment:
2.4 is in beta feature freeze so this is 2.5 now.

Reply:
Why 2.5?
This is a DB schema change it can go to 2.4.1 isnt it?

Reply:
I have updated that and put 2.4.1 for know. Let me know if this seems fine by you.

Reply:
@eladkal Because 2.4.x is for bug fixes only, and this is a new feature.

Reply:
Updated to 2.5.0, thanks :)